go.work.sum

```diff
@@ -0,0 +1,986 @@
+bitbucket.org/creachadair/shell v0.0.7 h1:Z96pB6DkSb7F3Y3BBnJeOZH2gazyMTWlvecSD4vDqfk=
```

Comment:
This file isn't up to date. If you remove this file and re-generate it (run `make build`), then you will see that lots of entries will be cleaned up.

Reply:
This is because while running `make build`, the `tools/` submodules are not touched. Then, it deletes those dependencies. Running `make update-go-workspace` reconciles the changes.